cspell.json

```diff
@@ -89,7 +89,9 @@
                 "tschüß",
                 "TSCHÜSS",
                 "ὈΔΥΣΣΕΎΣ",
-                "ὀδυσσεύς"
+                "ὀδυσσεύς",
+                "wgpu",
```

Comment:
shouldn't this go for all language and not only markdown?

Reply:
Oh, yes, I suppose so. I'll change that.